port/src/input.c

```diff
@@ -709,6 +710,18 @@ s32 inputInit(void)
 		SDL_SetHint(SDL_HINT_JOYSTICK_RAWINPUT_CORRELATE_XINPUT, "1");
 	}
 
+	if (useNintendoLayout) {
+		// use the Nintendo-style face button layout across both menus and gameplay
+    // this is default hint in SDL 2.0.6 and later
+		SDL_SetHint(SDL_HINT_GAMECONTROLLER_USE_BUTTON_LABELS, "1"); // uses button labels (A/B/X/Y)
+	} else {
+#if SDL_VERSION_ATLEAST(2, 0, 6)
```

Comment:
I'd say it's not worth adding checks for `SDL_VERSION`s below 2.0.9, they're pretty ancient, so you can remove this one.